google/cloud/spanner_dbapi/connection.py

```diff
@@ -126,6 +134,15 @@ def autocommit(self):
         """
         return self._autocommit
 
+    @property
+    def buffer_ddl_statements(self):
+        """Whether to buffer ddl statements for this connection.
```

Comment:
This needs more documentation. I (think I) understand what it means. To someone who is not familiar with Cloud Spanner, it is not clear what this means. It is also not clear when they should enable/disable this.